crates/autopilot/src/infra/persistence/dto/order.rs

```diff
@@ -281,8 +281,11 @@ pub enum FeePolicy {
 #[derive(Clone, Debug, Serialize, Deserialize)]
 #[serde(rename_all = "camelCase")]
 pub struct Quote {
+    #[serde_as(as = "HexOrDecimalU256")]
```

Comment:
If you changed this, then you also have to update the `driver` side: https://github.com/cowprotocol/services/blob/main/crates/driver/src/infra/api/routes/solve/dto/auction.rs#L329-L332

[Tests](https://github.com/cowprotocol/services/pull/2539) probably not catching this since they are not yet merged.
Driver tests apparently have their own (also wrong) serialization that needs to be updated: https://github.com/cowprotocol/services/blob/main/crates/driver/src/tests/setup/fee.rs#L33